jsapp/js/account/plans/addOnList.module.scss

```diff
@@ -0,0 +1,61 @@
+@use '~kobo-common/src/styles/colors';
+@use 'scss/breakpoints';
+@use 'scss/sizes';
+
+.header {
+  font-size: sizes.$r18;
+  font-weight: 700;
+  color: colors.$kobo-storm;
+  text-transform: uppercase;
+}
+
+.caption {
+  text-align: start;
+}
+
+.table {
+  table-layout: fixed;
+  border-collapse: separate;
+  border-spacing: 1em;
+
+  & td {
+    padding-block: 0.5em;
+    font-weight: 700;
+    overflow: hidden;
+  }
+}
+
+.row:first-child > td:not(:last-child) {
```

Comment:
Shouldn't this be `.row > td`?

Reply:
Ah, I think I know why this doesn't match the Figma - I've been working off of a mockup from @tinok for this phase of implementing add-ons:
<img width="878" alt="image" src="https://github.com/kobotoolbox/kpi/assets/7819986/2c677dce-8063-481e-805d-1e253675eb2a">

Reply:
From the Figma:
![image](https://github.com/kobotoolbox/kpi/assets/7819986/793ee633-b0a9-4c94-9132-463270f30d1e)

Reply:
I've updated it to match the Figma - might as well, since we'd have to do it later anyways 👍